src/index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
     <link rel="stylesheet" type="text/css" href="style.css">
-    <title>Activity Tracker</title>
```

Comment:
@Garrett-Iannuzzi  These got accidentally deleted so I've re-added them and updated my pull request.